EpikCord/client/http.py

```diff
@@ -2,14 +2,14 @@
 
 import asyncio
 import mimetypes
-from functools import partialmethod
 from importlib.util import find_spec
 from io import IOBase
 from logging import getLogger
 from typing import Any, Dict, List, Optional, Type, Union
 
+from .buckets import TopLevelBucket, Bucket, MockBucket
 from .. import __version__
-from ..exceptions import BadRequest, Forbidden, HTTPException, NotFound, Unauthorized
+from ..exceptions import BadRequest, Forbidden, HTTPException, NotFound, Unauthorized, TooManyRetries
 from ..utils import clear_none_values
 from .websocket import GatewayWebsocket
```

Comment:
This import is invalid

Reply:
I've defined it in exceptions.py, unless you mean something else.

Reply:
i am talking about `GatewayWebsocket`